lib/cloudfront/models/InvalidationConfig.rb

```diff
@@ -0,0 +1,22 @@
+module Cumulus
+  module CloudFront
+    # Public: An object representing configuration for a CloudFront invalidation
+    class InvalidationConfig
+      attr_reader :distribution_id
+      attr_reader :paths
+
+
+      # Public: Constructor
+      #
+      # json - a hash containing the JSON configuration for the invalidation
+      def initialize(name, json = nil)
+        if !json.nil?
+          @name = name
+          @distribution_id = json["distribution-id"]
```

Comment:
You could make this `json.fetch("distribution-id")` to force them to have to supply a distribution id.

Catch a KeyError and produce a readable error message, then exit?